src/table/cells/handle-cell-focus.js

```diff
@@ -0,0 +1,27 @@
+export const focusCell = (rowElements, nextCellCoord) => {
+  const nextCell = rowElements[nextCellCoord[0]].getElementsByClassName('sn-table-cell')[nextCellCoord[1]];
+  nextCell.focus();
+  nextCell.setAttribute('tabIndex', '0');
+};
+
+const resetTabIndex = (rowElements, focusedCellCoord) => {
+  const cell = rowElements[focusedCellCoord[0]]?.getElementsByClassName('sn-table-cell')[focusedCellCoord[1]];
+  cell && cell.setAttribute('tabIndex', '-1');
+};
+
+export const handleBodyCellFocus = (cell, focusedCellCoord, rootElement) => {
+  const { rawRowIdx, rawColIdx } = cell;
+  const rowElements = rootElement.getElementsByClassName('sn-table-row');
+  resetTabIndex(rowElements, focusedCellCoord.current);
+  const focusedCellCoordCopy = focusedCellCoord;
```

Comment:
Why is this needed?

Reply:
the lint in circleCI complains no-param-reassign. So did a fix here.

Reply:
Yeah we do that all over the place, so you can actually add an ignore comment here. Still keeping it as a rule since it's good to keep track of.

Reply:
would be good with some simple tests here, on for `handleBodyCellFocus` and one for `handleHeadCellFocus` should be enough

Reply:
> Yeah we do that all over the place, so you can actually add an ignore comment here. Still keeping it as a rule since it's good to keep track of.

I think it is better to do a fix like that instead of adding an ignore comment.

Reply:
> would be good with some simple tests here, on for `handleBodyCellFocus` and one for `handleHeadCellFocus` should be enough

Yes, I have been writing the unit testing, which will be committed later today.

Reply:
> > Yeah we do that all over the place, so you can actually add an ignore comment here. Still keeping it as a rule since it's good to keep track of.
> 
> I think it is better to do a fix like that instead of adding an ignore comment.

I don't. Pretty sure you are actually just creating a reference to the real `focusedCellCoord` anyway, otherwise `focusedCellCoord` wouldn't be updated where it is stored. So you still have the risk of unexpected behaviour when reassigning arguments, but the linter doesn't catch it :/ Could maybe make it throw warnings instead of errors though

Reply:
Actually, I think this is relevant. https://eslint.org/docs/rules/no-param-reassign#props So reassigning props is fine but not the entire argument. Feel free to set that up if you want :)

Reply:
Actually, I added it 🙃  https://github.com/qlik-oss/sn-table/pull/173

Reply:
> Actually, I added it 🙃 #173

Good. Thanks.